star_tides/core/actions/base_action.py

```diff
@@ -24,3 +24,6 @@ def run(self):
         ''' Must be implemented on every child class.
         '''
         pass
+
+    def get_caller_user_id(self):
+        return 'some-user' # FIXME
```

Comment:
This should be a static method in the `base_controller` because it has to do with the request. Add a class method to that file and call the `decode_jwt` method which will have the calling user's id.

Then you can invoke it like this:
```
Controller.get_calling_user_id()
```

Reply:
Done.